### DISCUSSION TITLE
Non-Docker Initial Setup - Unable to Access Admin Web

### DISCUSSION THREAD

**Discussion by smartin-mfmbp** [Keyfactor/signserver-ce#44] [Q&A]
After much searching, reading, and Youtube watching I have finally succumbed to having post a question.

As stated, this is an initial install in a VM - not a docker image. I have installed Wildfly 24, Postgresql 15, and deploying Signserver 5.11.1. I have an in-house step-ca server to issue certificates.
I have issued a server cert and can connect to Wildfly securely.
I have issued and imported a valid user certificate into my browsers.
I have allowed all admins with the `bin/signserver wsadmins -allowany` command.

On the admin web page, when I click on the Use TLS Certificate link I get an Authorization Error. I do not get a pop-up asking for my client certificate.
I'm guessing this has to do with some configuration that is not set correctly or not set at all.

Is Signserver using the same keystore that Wildfly is?
If not, where does the Root CA certificate need to be stored?
Where are users added?

I apologize if these questions are addressed in the documentation, but I have not found them there. If they are discussed, please let me know where, I hate posting questions if they are addressed in the documentation.

Thanks, 
Steve

**Comment by netmackan** [comment]
Hi Steve,

> On the admin web page, when I click on the Use TLS Certificate link I get an Authorization Error. I do not get a pop-up asking for my client certificate.
I'm guessing this has to do with some configuration that is not set correctly or not set at all.
> Is Signserver using the same keystore that Wildfly is?
If not, where does the Root CA certificate need to be stored?

Yes, this sounds like a configuration issue with the connector in WildFly or simply that the issuing CA is not in the trust store.
In our configuration instructions for WildFly you would setup the connectors so that there is keystore called standalone/configuration/keystore/truststore.jks and that would be that one that needs your issuing CA's certificate.

The instructions are here:
https://doc.primekey.com/signserver/signserver-installation/application-server-setup/wildfly-10+-and-jboss-eap-7-1+

However, sometimes it is just easier to look directly in standalone/configuration/standalone.xml to see if something is missing.
